freeipa/src/main/resources/freeipa-salt/salt/freeipa/scripts/freeipa_backup

```diff
@@ -93,25 +111,25 @@ elif [[ "$TYPE" = "DATA" ]]; then
 fi
 
 doLog(){
-    type_of_msg=$(echo $*|cut -d" " -f1)
+    type_of_msg=$(echo "$*"|cut -d" " -f1)
     msg=$(echo "$*"|cut -d" " -f2-)
-    [[ $type_of_msg == DEBUG ]] && [[ $do_print_debug_msgs -ne 1 ]] && return
+    [[ $type_of_msg == DEBUG ]] && [[ ${PRINT_DEBUG_MSGS} -ne 1 ]] && return
     [[ $type_of_msg == INFO ]] && type_of_msg="INFO " # one space for aligning
     [[ $type_of_msg == WARN ]] && type_of_msg="WARN " # as well
 
     # print to the terminal if we have one
-    test -t 1 && echo "`date "+%Y-%m-%dT%H:%M:%SZ"` $type_of_msg ""$msg"
-    echo "`date "+%Y-%m-%dT%H:%M:%SZ"` $type_of_msg ""$msg" >> $LOGFILE
+    test -t 1 && echo "$(date "+%Y-%m-%dT%H:%M:%SZ") $type_of_msg ""$msg"
+    echo "$(date "+%Y-%m-%dT%H:%M:%SZ") $type_of_msg ""$msg" >> "${LOGFILE}"
 }
 
 doStatus(){
     if [[ -n "$BACKUP_SCHEDULE" ]]
     then
-        type_of_msg=$(echo $*|cut -d" " -f1)
+        type_of_msg=$(echo "$*"|cut -d" " -f1)
         msg=$(echo "$*"|cut -d" " -f2-)
         [[ $type_of_msg == INFO ]] && type_of_msg="INFO " # one space for aligning
 
-        echo "`date "+%Y-%m-%dT%H:%M:%SZ"` $type_of_msg $msg" > $STATUSFILEPREFIX$BACKUP_SCHEDULE.log
+        echo "$(date "+%Y-%m-%dT%H:%M:%SZ") $type_of_msg $msg" > "${STATUSFILEPREFIX}""${BACKUP_SCHEDULE}".log
```

Comment:
Is there a reason to have double quotes nextto each other? `echo "$(date "+%Y-%m-%dT%H:%M:%SZ") $type_of_msg $msg" > "${STATUSFILEPREFIX}${BACKUP_SCHEDULE}".log` might be a little cleaner.

Reply:
fixed